x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/alert_summary_widget.tsx

```diff
@@ -23,6 +23,7 @@ export const AlertSummaryWidget = ({
   onClick = () => {},
   timeRange,
   hideChart,
+  onLoaded,
```

Comment:
I need this to report back to the embeddable when rendering is complete. The reporting plugin waits for a renderComplete signal when the embeddable has finished loading any data and is correctly rendered so the headless Chromium can take a screenshot. Related to [this](https://github.com/elastic/kibana/pull/169929)

Reply:
I am wondering if `onRenderComplete` is a better name. not a deal breaker.